File: PhotoMasterBackend/PhotoMasterBackend/DTOs/Photo.cs
using System;
using System.Collections.Generic;

namespace PhotoMasterBackend.DTOs
{
    public class Photo
    {
        public int Id { get; set; }

        public DateTime Date { get; set; }

        public string Path { get; set; }

        public string Location { get; set; }

        public string Description { get; set; }

        public List<Label> Labels { get; set; }
    }
}

File: PhotoMaster/src/app/Models/Photo.ts
import { Label } from './Label';

export class Photo {
    id: number;
    date: string;
    path: string;
    location: string;
    description: string;
    labels: Label[];

    constructor(id: number, date: string, location: string, description: string, labels: Label[]){
        this.id = id;
        this.date = date;
        this.location = location;
        this.description = description;
        this.labels = labels;
    }
}